calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| `radar_initial_crop_box_min_z`              | `double`      | `-50.0`                                                 | Minimum z-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_x`              | `double`      | `50.0`                                                  | Maximum x-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_y`              | `double`      | `50.0`                                                  | Maximum y-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_z`              | `double`      | `50.0`                                                  | Maximum z-coordinate for the initial radar calibration area.                                                                                      |
+| `lidar_background_model_leaf_size`          | `double`      | `0.1`                                                   | Voxel size in meter for the lidar background model.                                                                                               |
+| `radar_background_model_leaf_size`          | `double`      | `0.1`                                                   | Voxel size in meter for the radar background model.                                                                                               |
+| `max_calibration_range`                     | `double`      | `50.0`                                                  | Maximum range for calibration in meters.                                                                                                          |
+| `background_model_timeout`                  | `double`      | `5.0`                                                   | The background model will terminate if it is not updated within this period, measured in seconds.                                                 |
+| `min_foreground_distance`                   | `double`      | `0.4`                                                   | Square of this value in meters is used for filtering foreground points, typically needing to be at least double the `background_model_leaf_size`. |
+| `background_extraction_timeout`             | `double`      | `15.0`                                                  | Timeout in seconds for background extraction processes.                                                                                           |
+| `ransac_threshold`                          | `double`      | `0.2`                                                   | Distance threshold for the segmentation model.                                                                                                    |
```

Comment:
Unit

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/fd55de1142b05d40e886ef949233d043bbe9fedd